gravitee-apim-console-webui/src/management/api/general/details/api-general-info-duplicate-dialog/api-general-info-duplicate-dialog.component.ts

```diff
@@ -71,6 +72,26 @@ export class ApiGeneralInfoDuplicateDialogComponent implements OnDestroy {
         }, {}),
       ),
     });
+
+    if (dialogData.api.definitionVersion === 'V4') {
+      if (dialogData.api.listeners?.find((listener) => listener.type === 'TCP')) {
+        // FIXME: discuss during review
+        //  Should we reuse and mutualize what has been done in this PR ? https://github.com/gravitee-io/gravitee-api-management/pull/6298/files#diff-ca4491fa6ad94ef1e952b9c2e8a3bb7c354795e8c1a9783b93811cc9d23143e2R195-R204
+        //  or should we just rely on the backend response? as done for contextPath
+        //  My opinion is the better would be to have a sync validator in front side to prevent useless calls to the mAPI
+        this.duplicateApiForm.addControl('host', new UntypedFormControl('', [Validators.required], [this.apiV2Service.hostValidator({})]));
```

Comment:
What about something like [this](https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-console-webui/src/management/api/component/gio-form-listeners/gio-form-listeners-tcp-hosts/gio-form-listeners-tcp-hosts.component.ts#L208) ?

To be honest I'm not confortable about the usage of the service directly as a validator. Maybe we could export the validator in a dedicated file and make it look like a real validator (with the good signature, structure...)